provider/resource_notable_event_aggregation_policy.go

```diff
@@ -634,7 +654,7 @@ func ResourceNotableEventAggregationPolicy() *schema.Resource {
 func criteria(criteria_type string, criteria_data *schema.Set) (criteria map[string]interface{}, err error) {
 	criterias := criteria_data.List()
```

Comment:
not a big deal, but the word `criteria` is already plural, the singular noun is `criterion` :)